src/storage/secondary/column/primitive_column_builder.rs

```diff
@@ -145,14 +146,26 @@ impl<T: PrimitiveFixedWidthEncode> ColumnBuilder<T::ArrayType> for PrimitiveColu
                                 builder
                             )));
                     }
-                    (true, false) => {
+                    (true, EncodeType::Plain) => {
                         self.current_builder = Some(BlockBuilderImpl::PlainNullable(
                             PlainPrimitiveNullableBlockBuilder::new(
                                 self.options.target_block_size - 16,
                             ),
                         ));
                     }
-                    (false, true) => {
+                    (true, EncodeType::Dictionary) => {
+                        let builder = PlainPrimitiveNullableBlockBuilder::new(
+                            self.options.target_block_size - 16,
+                        );
+                        self.current_builder =
+                            Some(BlockBuilderImpl::RleNullable(RleBlockBuilder::<
```

Comment:
typo？

Reply:
Yes, I would mark it as todo and wait until ordered_float is supported